.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## Purpose
+
+Fixes:
```

Comment:
Lets remove this.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/121/commits/0c233e7841b2df50bb044754258a5d404821513a